app-tasks/dags/ingest/ingest_project_scenes.py

```diff
@@ -68,21 +77,36 @@ def execute_ingest_batch_job(ingest_s3_uri, ingest_def_id):
     Returns:
         dict
     """
-    batch = boto3.client('batch')
-    ingest_arn = get_latest_batch_job_revision()
-    response = batch.submit_job(
-        jobName='ingest-{}'.format(ingest_def_id),
-        jobQueue=batch_job_queue,
-        jobDefinition=ingest_arn,
-        parameters={
-            'ingest_definition': ingest_s3_uri
+    step = {
+        'ActionOnFailure': 'CONTINUE',
+        'Name': 'ingest-{}'.format(ingest_def_id),
+        'HadoopJarStep': {
+            'Args': ['/usr/bin/spark-submit',
+                     '--master',
+                     'yarn',
+                     '--deploy-mode',
+                     'cluster',
+                     '--class',
+                     'com.azavea.rf.batch.ingest.Ingest',
+                     '--driver-memory', '8G',
```

Comment:
I think the driver memory can be omitted here. It was included for AWS Batch because batch was running a spark local cluster and that was to control how much memory that would consume.